tests/api-watch-diff/dune

```diff
@@ -0,0 +1,6 @@
+(library
+ (name ppx_expect_test)
+ (inline_tests)
+ (preprocess
+  (pps ppx_expect))
+ (libraries ppx_expect.evaluator api_watch_diff))
```

Comment:
I don't think you need `ppx_expect.evaluator` here it's meant for the test runner which dune is generating by itself, not for the test file.

Reply:
Oh okay! Thanks for pointing it out. I will remove it.